b3lb/rest/templates/metrics_all

```diff
@@ -1,10 +1,19 @@
 # HELP b3lb_node_load Calculated node load
 # TYPE b3lb_node_load gauge
 {% for node, cluster, load in nodes %}b3lb_node_load{node="{{ node }}",cluster="{{ cluster }}"} {{ load }}
-{% endfor %}# HELP secret_endpoint_limit Attendee limit per secret
-# TYPE secret_endpoint_limit gauge
-{% for secret in secrets %}b3lb_endpoint_limit{secret="{{ secret }}"} 0
-{% endfor %}{% for metric_key, metric_value in metrics.items() %}# HELP b3lb_{{ metric_key }} {{ metric_value.name_choice }}
+{% endfor %}{% if tenant_limits %}# HELP b3lb_secret_tenant_limit Attendee limit per tenant
```

Comment:
Metrics naming suggestion:
- `b3lb_secret_attendee_limit{secret=...}`
- `b3lb_secret_meeting_limit{secret=...}`
- `b3lb_tenant_attendee_limit{secret=...}`
- `b3lb_tenant_meeting_limit{secret=...}`

`TYPE` hint is missing?

Reply:
Renamed metrics. `TYPE` already exists.